engines/cge2/talk.cpp

```diff
@@ -251,9 +258,18 @@ InfoLine::InfoLine(CGE2Engine *vm, uint16 w, ColorBank color)
 	V2D siz = V2D(_vm, w, kFontHigh);
 	b[0] = Bitmap(_vm, siz.x, siz.y, _color[2]);
 	setShapeList(b, 1);
+	_lastText = "";
```

Comment:
It would be better to do this initialisation in the constructor initializer list.

If you don't know what this means, see for example https://www.learncpp.com/cpp-tutorial/constructor-member-initializer-lists/